optuna/study.py

```diff
@@ -751,8 +751,12 @@ def _run_trial(
 
         return trial
 
-    def _log_completed_trial(self, trial, result):
-        # type: (trial_module.Trial, float) -> None
+    def _log_completed_trial(self, trial: trial_module.Trial, result: float) -> None:
+
+        if logging.get_verbosity() > logging.INFO:
```

Comment:
How about encapsulating this entire logic in `optuna.logging._is_enabled_for(logger, level)`, or something, since it's used in the MO study as well?